src/ProjectNeon/Assets/Features/Effects/EffectOnTurnStart.cs

```diff
@@ -0,0 +1,30 @@
+﻿using UnityEngine;
+using System.Collections;
+
+/**
+ * Effects that trigger on Turn Start.
+ */
+public class EffectOnTurnStart : Effect
+{
+    private Effect _effect;
+    private Member _performer;
+    private Target _effectTarget;
+
+    public EffectOnTurnStart(Effect origin)
+    {
+        _effect = origin;
+    }
+
+    public void Apply(Member source, Target target)
+    {
+        _performer = source;
```

Comment:
These should be passed in the lambda, rather than stored in the class.

`BattleEvent.Subscribe<TurnStart>(_ => Execute(source, target), this);`

Reply:
@SilasReinagel Done